src/Lti/Extractor/ReplaceResultSourceIdExtractor.php

```diff
@@ -39,7 +39,7 @@ public function extractSourceId(string $xmlContent): int
             throw new InvalidLtiReplaceResultBodyException();
         }
 
-        $xml->registerXPathNamespace('x', 'http://www.imsglobal.org/lis/oms1p0/pox');
+        $xml->registerXPathNamespace('x', 'http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0');
```

Comment:
We should also create a test for this: https://github.com/oat-sa/simple-roster/blob/develop/tests/Unit/Lti/Extractor/ReplaceResultSourceIdExtractorTest.php

Reply:
I know XML files were updated, but we could actually make clear in the test this namespace only should be considered. wdyt?

Reply:
It is already covered by `\App\Tests\Unit\Lti\Extractor\ReplaceResultSourceIdExtractorTest::testItCanExtractSourceId`. But I have added a unit test to ensure if an exception is thrown if xml has wrong namespace.
Good point